src/TaylorIntegration.jl

```diff
@@ -8,6 +8,7 @@ import Compat.view
 
 export  taylorinteg
 
-include("integration_methods.jl")
+include("jettransport.jl")
+include("explicitode.jl")
 
```

Comment:
Since `explicitode.jl` has the basic functions for taylor integration, I suggest that it appears as the first included file.